bramkouwen/Neatly-In-Motion
# Creating the Home Page's Main Content

## What does this PR do?
* It creates the main content for index.html 

## How do you check this PR?
* Pull down the branch
* Check the added content to index.html

## Screenshots 
None.
